### PR TITLE
fix(RichTextEditor): Update content when value property is updated

### PR DESCRIPTION
**Pull request by triggertoo**

### CODE REVIEW

**Review by Walkeezy — APPROVED** [review]

File: src/components/RichTextEditor/hooks/useEditorValueUpdates.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useRef } from "react";
+import { Editor, Transforms } from "slate";
+import { clearEditor } from "../utils/editor/clear";
+import { parseRawValue } from "../utils/editor/parseRawContent";
+
+/**
+ * Updates the editor value when the initival value changes
```
Comment (SamuelAlev) [inline src/components/RichTextEditor/hooks/useEditorValueUpdates.ts:9]: Suggestions is broken on GH for now 🤷 
There is a typo "when the initival value changes"

File: src/components/RichTextEditor/RichTextEditor.tsx [hunk 2]
```diff
@@ -71,6 +72,7 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
     const [{ matches, children }, send] = useMachine(() =>
         editorMachine.withContext({ locked: readonly, onTextChange, onBlur }),
     );
+    useEditorValueUpdates(editor, initialValue);
```
Comment (SamuelAlev) [inline src/components/RichTextEditor/RichTextEditor.tsx:75]: Is it updating the xstate's context? Seems like the `editor` object is not touched here

Comment (triggertoo) [reply src/components/RichTextEditor/RichTextEditor.tsx:75]: `editor` and `value` are not part of the xstate context (they are sent with the events)

File: src/components/RichTextEditor/hooks/useEditorValueUpdates.ts (hunk 1 above)
Comment (triggertoo) [reply src/components/RichTextEditor/hooks/useEditorValueUpdates.ts:9]: https://github.com/Frontify/arcade/pull/435

**Pull request merged**